src/errors.js

```diff
@@ -170,7 +170,15 @@ export class TransactionError extends ServerError {
    */
 
   get resultCodes () {
-    return this._resultCodes
+    return this._resultCodes.operations.reduce((resultCodes, item, index) => {
```

Comment:
you have broken backwards compatibility here

Reply:
ok